frontend/public/locales/en/explorer.json

```diff
@@ -1,3 +1,4 @@
 {
-    "name_of_the_view": "Name of the view"
+    "name_of_the_view": "Name of the view",
+    "delete_confirm_message": "Are you sure you want to delete this view? Deleting a view is irreversible and cannot be undone."
```

Comment:
@Rajat-Dabade : We should be showing the name of the view.

Reply:
Added name in delete model, but need to find a way to highlight it. Will do this in next pr.